notes/stories/film/film-scifi-1960s.st.txt

```diff
@@ -5494,6 +5494,50 @@ the scientist profession [Two scientists performed an intelligence boosting oper
+black-and-white based on the 1961 novel Solaris by Stanisław Lem. It was the
+first film adaptation of the novel.
+
+Synopsis: A crew of scientists on a space station attempt to understand an
+extraterrestrial intelligence, which takes the form of a vast ocean on the
+titular alien planet.
+
+:: Date
+1968-10-08
+
+:: References
+https://en.wikipedia.org/wiki/Solaris_(1968_film)
+
+:: Ratings
+5 <paul>
+
+:: Major Themes
+space station [The film is set on a research station orbiting the titular planet Solaris.]
+coping with memories of a lost love [The arrival of the Harey aboard the space station stirred up memories in Kris of his suicided lover of whom she was a duplicate.]
+coping with the death of a spouse [Kris introspected solemnly on his suicided lover, Harie, over the course of the film.]
+what if my thoughts became reality [The Solaris planet ocean somehow manifested a duplicate of Kris' suicided lover, Harey, into reality. The story follows Kris as his comes to accept Harey as a replacement for his departed lover.]
+tragic love [Kris and the Harie duplicate's love ended tragically when she ended her own existence against Kris' will. This she did because she came to believe that she could never return to Earth with Kris as Kris dearly wanted.]
+what if my personal identity came into question [The viewer is made to ponder to what extent was the Harey duplicate that the Solaris planet ocean conjured up was the the same as the Harey who'd killed herself some years prior. In particular, a point was made about the Harey duplicate mirroring Kris' expectation of her.]
+incompatible modes of communication [A point was made about the alien mind of Solaris being so different from the human mind that clear communication between it and the people on the space station was not possible.]
+
+:: Minor Themes
+interplanetary space travel [Kris' docking with the space station was the culmination of a 487 day flight from Earth to Alpha Aquarii in the spacecraft Prometheus.]
+speculative spaceship [Kris traveled to the Solaris space station in the spacecraft Prometheus.]
+suicide [It was revealed that Kris' friend Gibaryan had taken his own life by injecting himself with a lethal dose of "pernostol". It was revealed that Kris' former lover Harey had taken her own life.]
+remorse [Kris felt responsible for Harie's suicide on account that she took her own life after he walked out on her.]
+beating one's self up [Kris held him self responsible for his former lover Harey's suicide, and lamented that he'd walked out on her after she said she'd take her own life, if he did so.]
+living isolated from society [Snaut spoke in passing of how people on other space stations had gone made from the isolation.]
+human life choice [Kris, Snaut, and Sartorius, after some deliberation, resolved to risk their lives by staying aboard the space station to complete their scientific investigation rather than turn tail and flee back to Earth.]
```

Comment:
Sounds like a sacrifice for one's people no? I forget the story but wasn't the mission important for mankind? I guess if they were concerned about their careers it's indeed a life choice.

Reply:
As I recall, they did it in the name of science. So I guess it could be construed as a sacrifice to advance human knowledge, and mankind by extension. But we don't have **sacrifice for one's people** anymore, so changed it to a plain old **human self-sacrifice**.